orchestra/migrations/0018_auto_20151007_1451.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def update_projects(apps, schema_editor):
+    Project = apps.get_model('orchestra', 'Project')
+    WorkflowVersion = apps.get_model('orchestra', 'WorkflowVersion')
+
+    for project in Project.objects.all():
+        project.workflow_version = WorkflowVersion.objects.get(
+            slug=project.workflow_slug)
+        project.save()
+
+
+def update_tasks(apps, schema_editor):
+    Task = apps.get_model('orchestra', 'Task')
+    WorkflowStep = apps.get_model('orchestra', 'WorkflowStep')
+
+    for task in Task.objects.all():
+        task.step = WorkflowStep.objects.get(slug=task.step_slug)
+        task.save()
+
+
+def migrate_worker_certs(apps, schema_editor):
+    Certification = apps.get_model('orchestra', 'Certification')
+    WorkerCertification = apps.get_model('orchestra', 'WorkerCertification')
+
+    # Copy unscoped worker certifications to the new scoped objects.
+    unscoped_worker_certs = WorkerCertification.objects.filter(
+        certification__workflow__isnull=True,
+    )
+    for worker_cert in unscoped_worker_certs:
+        scoped_certs = Certification.objects.filter(
+            workflow__isnull=False,
+            slug=worker_cert.certification.slug,
+        )
+
+        # Since unscoped certifications are 'global', create a new worker_cert
+        # for all new scoped certifications
+        for cert in scoped_certs:
+            WorkerCertification.objects.create(
+                certification=cert,
+                worker=worker_cert.worker,
+                task_class=worker_cert.task_class,
+                role=worker_cert.role
+            )
+
+        # Delete worker certs we successfully copied.
+        # Necessary so we can make Certification.workflow non-nullable.
+        if scoped_certs.exists():
+            worker_cert.delete()
+
+    # Verify that we've migrated all unscoped worker certifications
+    unmigrated_certifications = (Certification.objects
+                                 .exclude(workercertification=None)
+                                 .filter(workflow__isnull=True))
+    if unmigrated_certifications.exists():
+        raise ValueError('Failed to migrate all worker certifications!')
+
+    # Delete unscoped certifications.
+    # Necessary so we can make Certification.workflow non-nullable.
+    Certification.objects.filter(workflow__isnull=True).delete()
```

Comment:
ALL OF THE DATABASES. Except I don't think Sqlite even has an ALTER COLUMN command.